src/shared/install.h

```diff
@@ -30,6 +30,7 @@ enum UnitFileChangeType {
         UNIT_FILE_UNLINK,
         UNIT_FILE_IS_MASKED,
         UNIT_FILE_IS_DANGLING,
+        UNIT_FILE_WARNING,
```

Comment:
WARNING is a bit too generic i think. we might start warning about other stuff one day after all, and then the name is already taken.

maybe:

`UNIT_FILE_NO_SUCH_HOOK` 

or so?

after all this is about hooking stuff into things that don#t exist